momo_modules/gsm_module/src/gsm_serial.c

```diff
@@ -27,6 +26,7 @@ void enable_serial()
 	SPBRGH = 0;
 	SPBRGL = 16;
 
+	CREN = 0;
```

Comment:
Why is this turned off then on? If there's a reason we should document it in a comment.